frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTaskProgress.vue

```diff
@@ -26,36 +26,28 @@
       <span>{{ totalResponded | formatNumber }}</span
       >/{{ progressTotal | formatNumber }}
     </div>
-    <base-progress
+    <BaseProgress
       re-mode="determinate"
       :multiple="true"
       :progress="(totalSubmitted * 100) / progressTotal"
       :progress-secondary="(totalDiscarded * 100) / progressTotal"
+      :color="itemColor(0)"
+      :color-secondary="itemColor(1)"
     >
-    </base-progress>
+    </BaseProgress>
     <div class="scroll">
       <ul class="metrics__list">
-        <!-- FIXME - loop over an array -->
-        <li>
-          <span class="color-bullet submitted"></span>
-          <label class="metrics__list__name">Submitted</label>
-          <span class="metrics__list__counter">
-            {{ totalSubmitted | formatNumber }}
-          </span>
-        </li>
-        <li>
-          <span class="color-bullet discarded"></span>
-          <label class="metrics__list__name">Discarded</label>
-          <span class="metrics__list__counter">
-            {{ totalDiscarded | formatNumber }}
-          </span>
-        </li>
-        <li>
-          <span class="color-bullet pending"></span>
-          <label class="metrics__list__name">Pending</label>
-          <span class="metrics__list__counter">
-            {{ totalPending | formatNumber }}
-          </span>
+        <li v-for="(status, index) in progressItems" :key="index">
+          <span
+            class="color-bullet"
+            :style="{ backgroundColor: status.color }"
+          ></span>
+          <label class="metrics__list__name">{{
+            status.name | capitalize
+          }}</label>
+          <span class="metrics__list__counter">{{
+            status.progress | formatNumber
+          }}</span>
```

Comment:
be carefull of null exception, if status.name/progress is null, then there will be an error